Updating Tutorial Notebook and README

thyneb19 opened the pull request.

thyneb19 committed:
Update demo syntax and README

Updated to use LuxSQLTable syntax and updated the readme

thyneb19 committed:
Update College_Demo.ipynb

Fixed syntax for creating a LuxSQLTable

thyneb19 committed:
Update Binder link in README

dorisjlee committed:
Update README.md

dorisjlee commented on notebooks/College_Demo.ipynb:
```diff
@@ -21,7 +21,10 @@
     "\n",
     "data = pd.read_csv('https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/college.csv')\n",
     "engine = create_engine(\"postgresql://testuser:testpass@localhost:5432/testdb\")\n",
-    "data.to_sql(name='college', con=engine, if_exists = 'replace', index=False)"
```
Can we place the data upload scripts into a separate file to be imported so that it is not confused with the Lux SQL statements?

dorisjlee commented on notebooks/College_Demo.ipynb:
```diff
@@ -21,7 +21,10 @@
     "\n",
```
Can we add a description at the top motivating the use of SQL as an execution backend and why its important? Similar to [this](https://lux-api.readthedocs.io/en/latest/source/advanced/executor.html).

thyneb19 committed:
Added Script to populate Postgres DB

Updated PostBuild to run the data uploading scripts

thyneb19 committed:
Update College_Demo.ipynb

Added introduction section to the top, updated  the notebook to not include code that uploads data to the  Postgres DB
